backend/controllers/image.js

```diff
@@ -0,0 +1,78 @@
+const Images = require('../models/image'); // import the Mongoose Image model Schmea from image.js
+const fs = require('fs'); // import the fs package
+// The fs package is used to modify the file system.
+
+exports.createImage = (req, res, next) => { // call the post method, which adds a route to the router object, to handle POST requests to the /api/stuff endpoint
+  const url = req.protocol + '://' + req.get('host'); // set the url constant to the protocol and host of the request
+  req.body.image = JSON.parse(req.body.image); // parse the request body
+  const image = new Image({ // create a new Image object, using the Image model
+    title: req.body.image.title, // set the title property of the Image object to the title property of the request body
+    description: req.body.image.description, // set the description property of the Image object to the description property of the request body
+    ImageUrl: url + '/Images/' + req.file.filename, // set the ImageUrl property of the Image object to the ImageUrl property of the request body
```

Comment:
There are fields I was not expecting here which I assume are carried over from things and stuff, e.g. price but I am not sure we need most of the fields.